### PR TITLE
Fixing CoreLayout padding overflow

### PR DESCRIPTION
**Pull request by m4dz**

### CODE REVIEW

File: mdx-layout-core/src/CoreLayout.scss [hunk 1]
```diff
@@ -11,11 +11,7 @@ main {
 }
 
 article {
-  margin-left: auto;
-  margin-right: auto;
   overflow: auto;
-  width: 100%;
```
Comment (gqio) [inline mdx-layout-core/src/CoreLayout.scss:17]: just remove the width : 100% should be enough

**Comment by gqio** [comment]
centered :)

**Comment by georges-gomes** [comment]
I prefer center.

image

**Comment by m4dz** [comment]
Awesome, thanks @mveres 🤩 🙏 !

**Pull request merged**